packages/web-core/openapi/spec_v2.yaml

```diff
@@ -1573,6 +1573,10 @@ components:
           type: boolean
         isConditionalMediationAvailable:
           type: boolean
+        isConditionalCreationAvailable:
```

Comment:
I would transmit more raw data here (so the whole clientCapabilites if available)